app/scripts/components/data-catalog/index.tsx

```diff
@@ -90,6 +89,7 @@ const prepareDatasets = (
         d.name.toLowerCase().includes(searchLower) ||
```

Comment:
Shouldn't there be `d.id.toLowerCase().includes(searchLower) ||` ?

Reply:
Great catch @hanbyul-here! Funnily enough this was working well, but it was a coincidence. Fixed!